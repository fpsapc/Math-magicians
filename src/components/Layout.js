import React from 'react';
// eslint-disable-next-line import/no-extraneous-dependencies
import { Link } from 'react-router-dom';

const Layout = () => (
  <nav>
    <h1>Math Magicians</h1>
    <ul>
      <li><Link to="/"><span>Home</span></Link></li>
      <li><Link to="/calculator"><span>Calculator</span></Link></li>
      <li><Link to="/quote"><span>Qoute</span></Link></li>
      <li>Calculator</li>
      <li>Quote</li>
    </ul>
  </nav>
);

export default Layout;
