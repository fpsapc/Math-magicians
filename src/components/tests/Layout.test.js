import React from 'react';
// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from 'react-test-renderer';
import { HashRouter } from 'react-router-dom';
import Layout from '../Layout';

it('render correctly and mathes the snapshot', () => {
  const tree = renderer.create(
    <HashRouter>
      {' '}
      <Layout />
    </HashRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
