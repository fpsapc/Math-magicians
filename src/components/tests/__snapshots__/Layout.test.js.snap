// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly and mathes the snapshot 1`] = `
Array [
  " ",
  <nav
    className="navBar"
  >
    <h1
      className="navHeading"
    >
      Math Magicians
    </h1>
    <ul
      className="navLinks"
    >
      <li>
        <a
          href="#/"
          onClick={[Function]}
        >
          <span>
            Home
          </span>
        </a>
      </li>
      <li>
        <a
          href="#/calculator"
          onClick={[Function]}
        >
          <span>
            Calculator
          </span>
        </a>
      </li>
      <li>
        <a
          href="#/quote"
          onClick={[Function]}
        >
          <span>
            Quote
          </span>
        </a>
      </li>
    </ul>
  </nav>,
]
`;
